lib/licensed/sources/gradle.rb

```diff
@@ -122,49 +138,51 @@ def configurations
         end
       end
 
-      def self.add_gradle_license_report_plugins_block(gradle_build_file)
-
-        if gradle_build_file.include? "plugins"
-          gradle_build_file.gsub(/(?<=plugins)\s+{/, " { id 'com.github.jk1.dependency-license-report' version '1.16'")
-        else
-
-          gradle_build_file = " plugins { id 'com.github.jk1.dependency-license-report' version '1.16' }" + gradle_build_file
-        end
+      def project
+        @project ||= config.dig("gradle", "project")
```

Comment:
A few questions about this new value:

- Is a default value needed?  What's the expected behavior if a user doesn't specify `project` in their configuration?  
- Is there support for a user to run licensed over every project in a multi-project build?  If so, how does a user specify that?

Reply:
After reflexion I thought that the `apps` configuration key was a great option for multi-project builds. 
I therefore removed the `project` key and updated the code. 

To run licensed over multiple projects who shares the same licensed configuration you could do:

```
apps:
   - source_path: path/to/subprojectA
   - source_path: path/to/subprojectB|
   

what do you think of that?

Reply:
👍 if that works, that's a great fit with the intended use case of `apps`.  If there's a well-known pattern for naming subprojects you can also use glob ptterns like 

```yaml
apps:
  - source_path: path/to/subproject*
  
# or even forego the `apps` key and define the source_path at the root using a glob pattern
source_path: path/to/subproject*
```